README.rst

```diff
@@ -294,6 +294,53 @@ Next, follow the instructions for Linux systems, as provided above. For example:
     cd NeMo
     ./reinstall.sh
 
+Pure Windows Installation (Experimental)
```

Comment:
I wonder if this is really necessary to hand the pinned requirements file. Furthermore, there's no guarantee this won't break in the future. Do we need to support native windows install for ASR ? If so it can go into ASR docs rather than main readme cause no other domains will work on windows anyway. (Maybe TTS?)

Reply:
Yes TTS as well, but if we put it in ASR doc no one will ever find it (Imho)